files/securedrop-user-dom0-rpm-key-reimport.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Reimport SecureDrop Release Signing Key into rpm database
+
+[Service]
+Type=oneshot
+
+# The pubkey name format is `gpg-pubkey-$VERSION-$CREATION_SEC_SINCE_UNIX_EPOCH`,
+# where $VERSION is the last 8 characters of the GPG key's fingerprint, and
+# $CREATIONDATE is the key creation date, expressed as
+# `date -d "1970-1-1 + $((0x$CREATION_UNIX_EPOCH)) sec"`
+ExecCondition=sh -c "rpm -q gpg-pubkey-7b22e6a3-646b9337"
+ExecCondition=sh -c "test -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation"
+
+# From systemd documentation:
+# when an ExecCondition= command exits with exit code 1 through 254 (inclusive),
+# the remaining commands are skipped and the unit is not marked as failed.
+ExecStart=sh -c "rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n' | grep 'SecureDrop Release Signing Key' | cut -f1 | xargs sudo rpm -e"
+ExecStart=sh -c "sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation"
```

Comment:
```suggestion
ExecStart=sh -c "rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation"
User=root
```

Reply:
Weird. I'm fine with this change but confused, since for me a service file in /etc/systemd/user can import the key with sudo.

Reply:
cfm and I were actually seeing the same behaviour, which is an initial early failure in journal and then a subsequent success. I had missed the initial failure. It's an ordering issue; the first time, the service tries to run too soon, before appropriate systemd targets are reached.  Have pushed a change that should address this.